### DISCUSSION TITLE
Is there a way to hide skipped commands?

### DISCUSSION THREAD

**Discussion by technicalpickles** [evilmartians/lefthook#467] [Q&A]
We have several commands that are pretty narrowly scoped. If we are making small changes, you see a bunch of skipped steps which isn't really useful for an end-user:

```
sorbet: (SKIP. NO FILES FOR INSPECTION)
stylelint: (SKIP. NO FILES FOR INSPECTION)
codeowners: (SKIP. NO FILES FOR INSPECTION)
authorization: (SKIP. NO FILES FOR INSPECTION)
svgo: (SKIP. NO FILES FOR INSPECTION)
syncpack: (SKIP. NO FILES FOR INSPECTION)
eslint: (SKIP. NO FILES FOR INSPECTION)
packwerk: (SKIP. NO FILES FOR INSPECTION)
rubocop: (SKIP. NO FILES FOR INSPECTION)
```

I've looked at [skip_output](https://github.com/evilmartians/lefthook/blob/master/docs/configuration.md#skip_output) and tried all the options, but they are still always shown.

I found the code around it, and it _seems_ like there isn't a way:

- https://github.com/evilmartians/lefthook/blob/7fead412a9803a6af40e31303b7f2f23bb3f91bb/internal/lefthook/runner/prepare_command.go#L37-L44
- https://github.com/evilmartians/lefthook/blob/7fead412a9803a6af40e31303b7f2f23bb3f91bb/internal/lefthook/runner/prepare_command.go#L78-L85

**Comment by mrexox** [comment]
Hey! Thank you for your PR. Check the latest release (1.3.10) with `skip_output: [skips]` setting. Should skip all skips :)
